.github/ISSUE_TEMPLATE/docs-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Documentation Request
+about: General request for the Observability Documentation Team.
+title: "[Request]"
+labels: Team:Docs
+assignees: ''
+projects: 513
+
+---
+
+## Description
+*(Describe what needs to be documented. Link to relevant issues welcome)*
+
+## Collaboration
+
```

Comment:
Not sure if this matters in GitHub, but there's an empty line here after the heading but not elsewhere.